Sources/eppo/EppoClient.swift

```diff
@@ -102,25 +102,32 @@ public class EppoClient {
         host: String? = nil,
         assignmentLogger: AssignmentLogger? = nil,
         assignmentCache: AssignmentCache? = InMemoryAssignmentCache(),
-        initialConfiguration: Configuration? = nil
+        initialConfiguration: Configuration? = nil,
+        pollingEnabled: Bool = false,
+        pollingIntervalMs: Int = PollerConstants.DEFAULT_POLL_INTERVAL_MS,
```

Comment:
We return 0 for jitter if the jitter param is <0. Should we have a similar check for polling interval > 0?

Reply:
Updated!
https://github.com/Eppo-exp/eppo-ios-sdk/blob/3c423602ad96caf6b6fa58abd77e1c86f86d9581/Sources/eppo/Poller.swift#L110
https://github.com/Eppo-exp/eppo-ios-sdk/blob/3c423602ad96caf6b6fa58abd77e1c86f86d9581/Sources/eppo/Poller.swift#L126